workshops/models.py

```diff
@@ -317,6 +317,13 @@ class Event(models.Model):
     latitude = models.FloatField(null=True, blank=True)
     longitude = models.FloatField(null=True, blank=True)
 
+    REPO_REGEX = re.compile(r'https://github\.com/(?P<name>[^/]+)/'
+                            r'(?P<repo>[^/]+)/?')
+    REPO_SCHEMA = 'https://github.com/{name}/{repo}/'
```

Comment:
`FORMAT` instead of `SCHEMA`.  (The latter makes me think of a database schema.)